miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/UserInfoBridgeWrapper.kt

```diff
@@ -0,0 +1,154 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import androidx.annotation.VisibleForTesting
+import com.google.gson.Gson
+import com.rakuten.tech.mobile.miniapp.js.ErrorBridgeMessage.NO_IMPL
+import com.rakuten.tech.mobile.miniapp.js.userinfo.Contact
+import com.rakuten.tech.mobile.miniapp.js.userinfo.TokenData
+import com.rakuten.tech.mobile.miniapp.js.userinfo.UserInfoBridgeDispatcher
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionCache
+import com.rakuten.tech.mobile.miniapp.permission.MiniAppCustomPermissionType
+import java.util.ArrayList
+
+@Suppress("TooGenericExceptionCaught")
+internal class UserInfoBridgeWrapper {
+    private lateinit var bridgeExecutor: MiniAppBridgeExecutor
+    private lateinit var customPermissionCache: MiniAppCustomPermissionCache
+    private lateinit var miniAppId: String
+    private var isMiniAppComponentReady = false
+    private lateinit var userInfoBridgeDispatcher: UserInfoBridgeDispatcher
+    private var isReady = false
+
+    internal fun setMiniAppComponents(
```

Comment:
We don't need to mention this function as `internal` additionally as the entire class is already `internal` 🤔

Reply:
Correct but no harm to be cautious 😄

Reply:
Yeah, no harm but we could write clean code ;)